### DISCUSSION TITLE
unraid install

### DISCUSSION THREAD

**Discussion by frapie** [Budibase/budibase#4181] [Help]
any docs or a docker container for unraid that works please.

**Comment by shogunpurple** [comment]
Hi @frapie - budibase uses docker-compose for its docker setup and networks several containers together, rather than a single container.

We do not ship anything specific for unraid - using dockerman or similar. But you should be able to install and run `docker-compose` on unraid itself, I would imagine.

This could be helpful:
https://forums.unraid.net/topic/91436-is-docker-compose-available-on-unraid/

Once you have compose set up, you can follow the instructions here:
https://docs.budibase.com/self-hosting/hosting-methods/docker-compose
